seed/seed.json

```diff
@@ -594,6 +594,52 @@
                 "platform": ["WINDOWS"]
             }
         },
+        {
+            "name": "BraveAds.ShowCustomAdNotificationOnWindowsStudy",
```

Comment:
looks like we already have a study with the same name for `release` channel. shouldn't we change it instead?
https://github.com/brave/brave-variations/blob/8f6da1c1078b6f04da1b34c1501b1b68d20ec0c4/seed/seed.json#L679-L705

Reply:
@goodov the three studies have different filters:
```
            "filter": {
                "min_version": "91.1.26.37",
                "max_version": "92.1.30.18",
                "channel": ["NIGHTLY", "BETA"],
                "platform": ["WINDOWS"]
            }


            "filter": {
                "min_version": "92.1.30.19",
                "min_os_version": "10.0.17134.*",
                "channel": ["BETA", "NIGHTLY"],
                "platform": ["WINDOWS"]
            }


            "filter": {
                "min_version": "92.1.30.19",
                "channel": ["RELEASE"],
                "platform": ["WINDOWS"]
            }
```

Reply:
Fixed